### PR TITLE
Make metrics uniform

### PR DESCRIPTION
**Pull request by nl-ria**
* Added 2 new metrics `health_check_up` to check whether the exporter can get a response from the healthcheck endpoint and
`health_check_ok` - Status of a health check, 1 is OK.

### CODE REVIEW

**Comment by nl-ria** [comment]
Can someone help me with the tests please? Not sure I understand some of them.

**Comment by Limess** [comment]
Did our chat/changing the buffer size sort this @nl-ria or did you still need a hand?

**Comment by nl-ria** [comment]
I think I fixed the tests, but would appreciate a second pair of 👀  to make sure I haven't diluted them and/or testing the wrong things.

File: internal/collectors/collectors.go [hunk 1]
```diff
@@ -89,57 +105,82 @@ func (collector *healthcheckCollector) sendValidity(ch chan<- prometheus.Metric,
...
+			check.ID,
+			check.Name,
+			check.Severity.String(),
+			metric.System,
+		)
 	}
 }
 
-func (c *healthcheckCollector) sendInvalidMetric(ch chan<- prometheus.Metric, err error) {
+func (collector *healthcheckCollector) sendHealthcheckDownMetric(ch chan<- prometheus.Metric, err error) {
```
Comment (Limess) [inline internal/collectors/collectors.go]: Worth noting that it's idiomatic in Go to use short variable names: https://golang.org/doc/effective_go.html#interface-names

I don't particularly like this but it's something you might come across in future.

Comment (nl-ria) [reply internal/collectors/collectors.go]: Ah, but it wasn't uniform across. Don't mind using either `c` or `collector` as long as we use it throughout. Most of the functions here used `collector` so decided to go with that. Will keep this in mind tho !

**Pull request merged**